src/overlays/actors/ovl_En_Wonder_Talk/z_en_wonder_talk.h

```diff
@@ -6,9 +6,21 @@
 
 struct EnWonderTalk;
 
+typedef void (*EnWonderTalkFunc)(struct EnWonderTalk*, GlobalContext*);
+
 typedef struct EnWonderTalk {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x1C];
+    /* 0x014C */ EnWonderTalkFunc actionFunc;
+    /* 0x0150 */ s16 unk_150;
+    /* 0x0152 */ s16 unk_152;
+    /* 0x0154 */ s16 unk_154;
```

Comment:
```suggestion
    /* 0x0154 */ s16 switchFlag;
```
Might make sense to name it switchFlag? Not 100% sure

Reply:
Yup, I think it make sense